packages/node_modules/@ciscospark/plugin-memberships/src/notes-on-moderator-status.txt

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Please delete this file.

Reply:
@jpjpjp please delete this file from `src`. If you'd like to retain the information, a comment in the code or a spot in `docs` is more appropriate. Thanks!